Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -228,6 +246,13 @@ void MaxEnt::exec() {
   // Number of X bins
   const size_t npointsX = inWS->isHistogramData() ? npoints + 1 : npoints;
 
+  // Default target chi squared = the number of data points:
+  // it is what is expected from a good fit with a smooth function
+  // (given that the error values are correct)
+  if (chiTarget <= 1.0e-12) {
```

Comment:
@Anders-Markvardsen  here the default value is replaced with N points. I still think this is unclear for a user, but it does happen